### DISCUSSION TITLE
GitHub Pages creating infinite deploy jobs

### DISCUSSION THREAD

**Discussion by ghost** [community/community#47963] [Actions] [answered]
### Select Topic Area

Bug

### Body

![image](https://user-images.githubusercontent.com/125566061/219952838-a24e10cf-5e44-4616-a9b4-0e280e4786a2.png)
I have a static site [here](https://github.com/bsnts/site) that I'm trying to deploy with GitHub Pages. However, when I try and deploy it, GitHub spawns a bunch of deploy jobs, and creates a new one every time I visit the page. Nothing happens, and it sort of remains stuck like this indefinitely.

**Comment by ghost** [comment]
I've disabled actions for now, so hopefully it won't create any new jobs. The issue still stands, though.

**Comment by sunaurus** [comment]


**Reply by ghost** [reply]
Ah, yeah, there's performance issues being reported on https://www.githubstatus.com now.

**Comment by melvincwng** [comment, marked as answer]
Hey @bsnts :wave:, there seems to be issues with PRs and GitHub Actions right now - https://www.githubstatus.com/. 

The GitHub team is investigating and looking into it right now. 

In the meanwhile, please kindly wait for their updates and fixes by referring to the link above.

![20230219_221959](https://user-images.githubusercontent.com/77479885/219954096-3a55d955-d61d-483e-9cb9-b200475f027b.jpg)
